src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -860,12 +905,14 @@ class QualifyClauseSegment(BaseSegment):
     https://docs.snowflake.com/en/sql-reference/constructs/qualify.html
     """
 
-    type = "having_clause"
+    type = "qualify_clause"
```

Comment:
Fixing small mistake. Original author likely initially copied this segment from HAVING and forgot to update this.